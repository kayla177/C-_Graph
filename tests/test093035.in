ENTITY D8LGTJVJ name_D8LG paper
ENTITY 2YVERNHB name_2YVE institution
ENTITY YSDHO01T name_YSDH author
ENTITY 17OSHD8E name_17OS conference
ENTITY 99MOP085 name_99MO paper
ENTITY PDV3HUXE name_PDV3 author
ENTITY AZEZNMAF name_AZEZ conference
ENTITY 2JO8HN0J name_2JO8 conference
ENTITY 5DLCHLAY name_5DLC paper
ENTITY ZAUWRRSU name_ZAUW journal
RELATIONSHIP PDV3HUXE collaborated_with AZEZNMAF 7.3
RELATIONSHIP 2YVERNHB published_in ZAUWRRSU 2.6
RELATIONSHIP PDV3HUXE published_in 5DLCHLAY 4.2
RELATIONSHIP ZAUWRRSU authored_by 17OSHD8E 6.0
RELATIONSHIP PDV3HUXE authored_by 99MOP085 3.7
RELATIONSHIP ZAUWRRSU authored_by PDV3HUXE 4.5
RELATIONSHIP 2JO8HN0J cited_by ZAUWRRSU 6.6
RELATIONSHIP 2JO8HN0J collaborated_with 2YVERNHB 8.8
RELATIONSHIP D8LGTJVJ published_in YSDHO01T 0.9
RELATIONSHIP 2JO8HN0J published_in PDV3HUXE 4.2
RELATIONSHIP ZAUWRRSU affiliated_with YSDHO01T 4.1
RELATIONSHIP 2JO8HN0J published_in AZEZNMAF 6.1
RELATIONSHIP YSDHO01T presented_at AZEZNMAF 6.5
RELATIONSHIP PDV3HUXE affiliated_with YSDHO01T 7.9
RELATIONSHIP 2JO8HN0J presented_at 5DLCHLAY 3.3
HIGHEST
HIGHEST
DELETE ZAUWRRSU
DELETE 99MOP085
PRINT 5DLCHLAY
HIGHEST
PATH YSDHO01T AZEZNMAF
HIGHEST
PATH PDV3HUXE YSDHO01T
DELETE 5DLCHLAY
EXIT
