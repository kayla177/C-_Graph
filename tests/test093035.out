success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
D8LGTJVJ 2YVERNHB 22.3
D8LGTJVJ 2YVERNHB 22.3
success
success
PDV3HUXE 2JO8HN0J
D8LGTJVJ 2YVERNHB 22.3
YSDHO01T AZEZNMAF 6.5
D8LGTJVJ 2YVERNHB 22.3
PDV3HUXE YSDHO01T 7.9
success
